html/canvas/color.window.js

```diff
@@ -24,19 +24,19 @@ const context = canvas.getContext("2d");
   },
   {
     "input": "color(srgb 1 0 1)",
-    "output": "#ff00ff"
+    "output": "color(srgb 1 0 1)"
   },
   {
     "input": "color(srgb 1 0 1 / 1)",
-    "output": "#ff00ff"
+    "output": "color(srgb 1 0 1)"
   },
   {
-    "input": "color(srgb 1 0 1 / 0)",
-    "output": "rgba(255, 0, 255, 0)"
+    "input": "color(srgb 1 0 1/0)",
+    "output": "color(srgb 1 0 1 / 0)"
   },
   {
     "input": "color(srgb 1 none 1)",
-    "output": "color(srgb 1 none 1)" // one could argue for #ff00ff here?
+    "output": "color(srgb 1 none 1)"
```

Comment:
Does that preserve the `none` or is it converted to `0` on serialization? @weinig

Reply:
The specification states:

> For syntactic forms which support [missing color components](https://drafts.csswg.org/css-color-4/#missing-color-component), the value [none](https://drafts.csswg.org/css-color-4/#valdef-color-none) (equivalently NONE, nOnE, etc), shall be serialized in all-lowercase as the string "none".

Reply:
What I could maybe add here is a color such as `rgb(300 none 400)` which I guess should become `rgb(300 none 400)`?

Reply:
For my test: https://github.com/w3c/csswg-drafts/issues/10550#issuecomment-2291042801

For `color(srgb 1 none 1)`: both Chromium and WebKit preserve `none` here and that seems to be in line with the specification so I suggest we keep that as-is.